### PR TITLE
Update Owners and Owners-aliases

### PR DESCRIPTION
**Pull request by VaishnaviHire**
## Description
To expedite PR reviews and testing, added appropriate component owners to the repo. 

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    dashboard:
+    - andrewballantyne
+    - lucferbux
+    datasciencepipelines:
+    - HumairAK
+    modelregistry:
+    - dhirajsb
+    servicemesh:
+    - bartoszmajsak
+    - cam-garrison
```
Comment (bartoszmajsak) [inline OWNERS_ALIASES:26]:
```suggestion
    - aslakknutsen
    - bartoszmajsak
    - cam-garrison
```

WE ARE ABC!

**Comment by bartoszmajsak** [comment]
@VaishnaviHire how would it work now? Would it automatically add all people from `platform` alias for every PR review? That would be a huge load on your side.

On a semi-related note - what is the process for reviewing PRs? Specifically where I'm looking for clarification is:

- how many `lgtm`s does a PR need (how many reviewers) to be considered mergeable?
- who is responsible for merging the PR? clearly bot is not active (but maybe this will change it). should it be the author?
- can I change automatically assigned reviewers to those e.g. suggested by GH (who most recently touched given code) or me wanting to share this knowledge with people I think are most affected or have the most expertise in the area

And a side quest to consider:

- I am using [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) prefixes (or some subset of it), but there's no rule about it. Wondering if we want to leave it open or ask to have PR titles at least following this in this repository.

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    - LaVLaS
+    - VaishnaviHire
+    - AjayJagan
+    - ykaliuta
+    - ajaypratap003
+    - mattmahoneyrh
+    - asanzgom
+    serving:
+    - VedantMahabaleshwarkar
+    - israel-hdez
```
Comment (zdtsw) [inline OWNERS_ALIASES:14]:
```suggestion
    - israel-hdez
    - danielezonca
```

**Review by zdtsw — APPROVED** [review]

File: OWNERS_ALIASES [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    - danielezonca
+    ide:
+    - harshad16
+    dashboard:
+    - andrewballantyne
+    - lucferbux
+    datasciencepipelines:
```
Comment (anishasthana) [inline OWNERS_ALIASES:21]: @HumairAK should we have at least one more owner?

File: OWNERS_ALIASES [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    - AjayJagan
+    - ykaliuta
+    - ajaypratap003
+    - mattmahoneyrh
+    - asanzgom
+    serving:
+    - VedantMahabaleshwarkar
+    - israel-hdez
+    - danielezonca
+    ide:
```
Comment (anishasthana) [inline OWNERS_ALIASES:16]: @harshad16  should we have at least one more owner?

File: OWNERS_ALIASES [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    - harshad16
+    dashboard:
+    - andrewballantyne
+    - lucferbux
+    datasciencepipelines:
+    - HumairAK
+    modelregistry:
```
Comment (anishasthana) [inline OWNERS_ALIASES:23]: @dhirajsb should we have at least one more owner?

File: OWNERS_ALIASES (hunk 3 above)
Comment (HumairAK) [reply OWNERS_ALIASES:21]: yeah, add @gmfrasca

File: OWNERS_ALIASES [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    - dhirajsb
+    servicemesh:
+    - aslakknutsen
+    - bartoszmajsak
+    - cam-garrison
+    distributedworkloads:
+    - dimakis
+    aiexplainability:
```
[the remark this answers is not in the document]
Comment (ruivieira) [reply OWNERS_ALIASES:31]: Could @RobGeada be added too, please? Thanks!

File: OWNERS_ALIASES (hunk 5 above)
Comment (VaishnaviHire) [reply OWNERS_ALIASES:23]: @dhirajsb Can you recommend anyone from model registry that we can add as reviewers?

File: OWNERS_ALIASES [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+    - ykaliuta
+    - ajaypratap003
+    - mattmahoneyrh
+    - asanzgom
+    serving:
+    - VedantMahabaleshwarkar
+    - israel-hdez
+    ide:
+    - harshad16
+    - atheo89
```
Comment (anishasthana) [inline OWNERS_ALIASES:17]: Nit: Not in alphabetical order :-)

File: OWNERS_ALIASES [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+    - etirelli
+    - LaVLaS
+    - VaishnaviHire
+    - AjayJagan
+    - ykaliuta
+    - ajaypratap003
+    - mattmahoneyrh
+    - asanzgom
+    serving:
+    - VedantMahabaleshwarkar
```
Comment (anishasthana) [inline OWNERS_ALIASES:13]: Nit: Not in alphabetical order :-)

**Review by anishasthana — COMMENTED** [review]
Nit: Quite a few of the rows are not sorted in alphabetical order.

File: OWNERS_ALIASES [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+    - lucferbux
+    datasciencepipelines:
+    - HumairAK
+    modelregistry:
+    - dhirajsb
+    servicemesh:
+    - aslakknutsen
+    - bartoszmajsak
+    - cam-garrison
+    distributedworkloads:
```
[the remark this answers is not in the document]
Comment (anishasthana) [reply OWNERS_ALIASES:30]: @VaishnaviHire You can add @astefanutti as an owner for now.

File: OWNERS_ALIASES (hunk 5 above)
Comment (dhirajsb) [reply OWNERS_ALIASES:24]: @tarilabs and @rareddy can be added as model-registry reviewers.

**Review by harshad16 — COMMENTED** [review]
/lgtm
from my end.

**Pull request merged**